docs/guide/usage.md

```diff
@@ -29,6 +29,36 @@ app.use(Kongponents)
 app.mount('#app')
 ```
 
+### Using in Nuxt
+
+All components are fully SSR-compatible so there is no extra configuration needed for using Kongponents in Nuxt.
```

Comment:
we'll need to modify this a bit; there are some components that are definitely not compatible 
```suggestion
The majority of components are SSR-compatible so there is no extra configuration needed for using Kongponents in Nuxt or a server-side rendered project.
```